js/game/Director.js

```diff
@@ -502,4 +502,88 @@ function Director(scene){
 	// Goof.
 	self.update = function(){};
 
-}
\ No newline at end of file
+}
+
+///////////////////////////////////
+///// TEXT STRINGS ////////////////
+///////////////////////////////////
+
+var textStrings_EN = {
+    // Act 1
+    "niceHat"              : "OOH NICE HAT"                                          ,
+    "outtaHere"            : "yeah git on outta here"                                ,
+    "getARoom"             : "GROSS, GO GET A ROOM"                                  ,
+    "notCoolAnymore"       : "nvm hats aren't cool anymore"                          ,
+    "tvOnTv"               : "A TV... ON TV!"                                        ,
+    "cricky"               : "LIL' CRICKY <3"                                        ,
+    "tooManyCrickets"      : "okay that's too many crickets"                         ,
+    "normalPeep"           : "just a normal peep"                                    ,
+    "normalPeeps"          : "just some normal peeps"                                ,
+    "wowNothing"           : "WOWWEE, IT'S NOTHING"                                  ,
+    
+    // Act 2
+    "crazySquareAttacks"   : "CRAZED SQUARE ATTACKS"                                 ,
+    "justMissed"           : "oooooh just missed it"                                 ,
+    "somethingInteresting" : "(ya gotta catch 'em doing *something* interesting...)" ,
+    "whoIsScreaming"       : "(ya gotta catch who's screaming at 'em)"               ,
+    "circleFearsSquares"   : "CIRCLE FEARS SQUARES"                                  ,
+    "whoScaresThem"        : "(ya gotta also catch *who* they're scared by)"         ,
+    "areTheyScared"        : "(ya gotta catch 'em *being* scared by a square)"       ,
+    "squaresSnubCircles"   : "SQUARES SNUB CIRCLES"                                  ,
+    "areTheySnubbed"       : "(ya gotta catch 'em *while* snubbing a circle)"        ,
+    "everyoneHates"        : "EVERYONE HATES EVERYONE!!1!"                           ,
+    "almostEveryoneHates"  : "ALMOST EVERYONE HATES EVERYONE..."                     ,
+    "squaresHateCircles"   : "SQUARES HATE CIRCLES"                                  ,
+    "circlesHateSquares"   : "CIRCLES HATE SQUARES"                                  ,
+    "areTheyYelling"       : "(ya gotta catch 'em *yelling* at others)"              ,
+    "nerdsNow"             : "what are these nerds doing now"                        ,
+    "schockedPeep"         : "why's this peep shocked?"                              ,
```

Comment:
I don't think "peep" can be used in the singular. At least I've never heard of "a peep". Normally, we say "peeps". I'd say "dude" for the singular.